### ISSUE TITLE
Qt configuration does not seem respected on Linux

### ISSUE THREAD

**Issue by shane-kerr** [shane-kerr/Mintcoin-Desktop-Wallet#4]
It looks like neither "maxconnections" nor "addnode" are respected on Linux in the `~/.config/MintCoin/MintCoin-Qt.conf` file. Research and fix!

**Comment by shane-kerr** [comment]
It turns out that "maxconnnections" and "addnode" are configuration that is applicable to both the command-line daemon and the GUI, so are stored in `~/.MintCoin/MintCoin.conf`. The `~/.config/MintCoin/MintCoin-Qt.conf` file is only for GUI-specific options, although confusingly the GUI program will leave a single assignment of every option that it does not understand in there, so:
```
addnode=1.2.3.4
addnode=2.3.4.5
addnode=3.4.5.6
```
Will become:
```
addnode=3.4.5.6
```
And the others removed, without warning. 

I guess this is probably okay, and not worth fixing as this behavior falls out of the Qt configuration model. But maybe some documentation explaining it would be in order.

**Comment by shane-kerr** [comment]
Okay, this is addressed in the doc-updates branch:

https://github.com/shane-kerr/Mintcoin-Desktop-Wallet/tree/doc-updates

**Comment by shane-kerr** [comment]
I think this one is sufficiently addressed in the doc-updates and on my 2.0 branch.
